addons/web/static/src/views/fields/properties/properties_field.xml

```diff
@@ -27,6 +28,7 @@
                     </div>
                     <div class="o_property_field_value align-top w-100">
                         <PropertyValue
+                            id="'property-' + propertyConfiguration.name"
```

Comment:
Same question about collision and uniqueness.

If we consider this as an "id", we need to make sure it's going to be unique I think.
And if we allow people to specify their own property names, it could no-longer be the case.

Unless you put some guards to ensure uniqueness of the "name" at all times?

Reply:
properties can't have the same name, but it cost almost nothing to use an UUID instead if you prefer